src/OnboardingSPA/components/Drawer/DrawerPanel/DesignColors.js

```diff
@@ -54,6 +141,14 @@ const DesignColors = () => {
 		}
 	}
 
+	/**
+	 * Converts the user selected value into a suitable valid global styles array value
+	 *
+	 * @param {Object} selectedColorsLocalTemp - Color type mapped to the color
+	 *                                         e.g. {base: "#ffffff", contrast: "#404040", ... }
+	 * @param {string} colorStyle              - Selected Color Palette slug
+	 *                                         e.g. [{color: '#ffffff', name: 'Base', slug: 'base'}, ...]
+	 */
 	function LocalToState( selectedColorsLocalTemp, colorStyle ) {
```

Comment:
1. Camel case? 
2. Should localToState be setting `currentData`? From the description, it seems like it should just convert the value?
3. What is the difference between `local` and `state`?
4. Move to utils and avoid state change here?

Reply:
So basically state refers to the redux store value that would be saved. So basically the original `state` variable is very limited due to which we created a local version that is basically a smaller and more efficient version of the same, and the function localToState aims to convert it back into the original form after doing faster computations and saving it back to the **state**, hence Local to State.
Again being very specific to the colors step and won't moving it to utils just disperse the files thus reduce readability?